qiskit_dynamics/solvers/solver_functions.py

```diff
@@ -84,8 +95,15 @@ def solve_ode(
     - ``scipy.integrate.solve_ivp`` - supports methods
       ``['RK45', 'RK23', 'BDF', 'DOP853', 'Radau', 'LSODA']`` or by passing a valid
       ``scipy`` :class:`OdeSolver` instance.
-    - ``jax.experimental.ode.odeint`` - accessed via passing
-      ``method='jax_odeint'``.
+    - ``'RK4'``: A fixed-step 4th order Runge-Kutta solver.
```

Comment:
What is the difference between RK4 and RK45? Is the scipy RK45 not fixed step?
I wonder if we should use some naming convention so you can tell which methods are fixed step and which aren't.

Reply:
Yeah RK4 is a fixed step method, but RK45 is a variable step method that I believe uses 4th order runge kutta to take steps, but also computes 5th order runge kutta and compares to 4th order as part of it's routine for adjusting step size.

Turns out it's possible to actually "hack" the scipy RK45 with kwargs to behave like RK4, by just setting the tolerances insanely large and setting the `max_step` to whatever step size you want (described in comments [here](https://stackoverflow.com/questions/54494770/how-to-set-fixed-step-size-with-scipy-integrate)). I initially thought we could maybe drop the `RK4` method because of this, though `solve_ivp` will still be performing error checkign computations, which serve no purpose in this case, so maybe it's still worth keeping it.